frontend/src/css/_inputs.css

```
@@ -29,17 +35,98 @@
+  display: flex;
+  align-items: center;
+  background-color: var(--background);
+  border: 1px solid var(--surfaceSecondary);
+  box-shadow:
+    0 1px 1px hsl(0deg 0% 0% / 0.075),
+    0 2px 2px hsl(0deg 0% 0% / 0.075),
+    0 4px 4px hsl(0deg 0% 0% / 0.075),
+    0 8px 8px hsl(0deg 0% 0% / 0.075),
+    0 16px 16px hsl(0deg 0% 0% / 0.075);
+  padding: .25em .5em;
+  -webkit-box-align: center;
```

Stylelint (17.13.0)

[error] Expected "-webkit-box-align" to be "align-items" (property-no-deprecated)

(property-no-deprecated)